src/main/java/seedu/eduke8/question/QuestionList.java

```diff
@@ -70,7 +68,7 @@ public QuestionInterface getNextQuestion() {
     }
 
     @Override
-    public Boolean allQuestionsAnswered() {
+    public boolean areAllQuestionsAnswered() {
```

Comment:
Oh right! My mistake on the primitive versus object type for the boolean variable! Thanks for helping me edit that!
I think the new naming you have here is much clearer too!